protoc-gen-grpc-gateway/main_test.go

```diff
@@ -6,24 +6,26 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/v2/internal/descriptor"
 )
 
+// TODO(asnyder): These tests are covering a function parseFlags() which became unused after https://github.com/grpc-ecosystem/grpc-gateway/pull/1756. The function and tests may be deleted.
```

Comment:
Lets just have them out now if they're not adding any value!

Reply:
And they're gone! 🔥